features/lege-waarden.feature

```diff
@@ -0,0 +1,171 @@
+      | naam                | waarde                                  |
+      | type                | RaadpleegMetBurgerservicenummer         |
+      | burgerservicenummer | 555550002                               |
+      | fields              | burgerservicenummer,reisdocumentnummers |
+      Dan heeft de response een persoon met alleen de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 555550002 |
+  
+  Rule: Een object (groep) wordt geleverd als leeg object wanneer er ten minste één gegeven in die groep bestaat, maar geen enkel veld in die groep geleverd moet worden
+    - Hierbij geldt ook een veld dat niet met fields is gevraagd voor het bestaan van de groep
+    - hierbij geldt ook een veld met een standaardwaarde, leegwaarde of boolean false voor het bestaan van de groep
+ 
+    Scenario: Kind naam is onbekend geeft naam als leeg object
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 555550003 |
+      En de persoon heeft een 'kind' met alleen de volgende 'naam' gegevens
+      | naam          | waarde |
+      | geslachtsnaam | .      |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 555550003                       |
+      | fields              | kinderen.naam                   |
+      Dan heeft de response een persoon met een 'kind' met een leeg 'naam' object
+      # "kinderen": [{ "naam": {} }]
+
+    Scenario: Kind heeft geen voorvoegsel en alleen vragen om voorvoegsel geeft naam als leeg object
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 555550003 |
+      En de persoon heeft een 'kind' met alleen de volgende 'naam' gegevens
+      | naam          | waarde  |
+      | voornamen     | William |
+      | geslachtsnaam | Postma  |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 555550003                       |
+      | fields              | kinderen.naam.voorvoegsel       |
+      Dan heeft de response een persoon met een 'kind' met een leeg 'naam' object
+      # "kinderen": [{ "naam": {} }]
+
+    Scenario: Kind met geboorteplaats onbekend geeft geboorte als leeg object zonder plaats veld
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 555550003 |
+      En de persoon heeft een 'kind' met alleen de volgende 'geboorte' gegevens
+      | naam   | waarde   |
+      | datum  | 20070421 |
+      | plaats | 0000     |
```

Comment:
Hier moet of `plaats.code` of `geboorteplaats (03.20)`

Reply:
aangepast